modules/lazyload/module.php

```diff
@@ -124,7 +108,30 @@ public function __construct() {
 		} );
 
 		add_action( 'wp_footer', function() {
-			$this->enqueue_scripts();
+			?>
+			<script type='text/javascript' defer>
+				document.addEventListener( 'DOMContentLoaded', function() {
```

Comment:
Why not in a file that will be cached + import to the main file after the experiment will be merged?

Reply:
Rotem had the idea to load this code snippet inline to improve performance.